packages-exp/auth-exp/test/integration/webdriver/persistence.test.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { UserCredential } from '@firebase/auth-exp';
+import { expect } from 'chai';
+import { API_KEY } from '../../helpers/integration/settings';
+import { AnonFunction, PersistenceFunction } from './util/functions';
+import { browserDescribe } from './util/test_runner';
+
+browserDescribe('WebDriver persistence test', driver => {
```

Comment:
Side note: we should write some tests for the persistence events and cross-tab/cross-browser synchronization (in a different PR) b/182567813